packages/voyager-keycloak/src/KeycloakToken.ts

```diff
@@ -0,0 +1,34 @@
+// This function is taken directly from Keycloak's connector
+// here: https://github.com/keycloak/keycloak-nodejs-connect/blob/master/middleware/auth-utils/token.js#L30-L47
```

Comment:
Best to use commits instead of master as master will move and lines will change.

Reply:
Ignore that, but leaving just FYI